lib/lightning_web/live/job_live/manual_run_component.ex

```diff
@@ -60,25 +60,24 @@ defmodule LightningWeb.JobLive.ManualRunComponent do
         phx-submit="run"
       >
         <.dataclip_selector form={f} phx-target={@myself} dataclips={@dataclips} />
-        <div :if={is_nil(@selected_dataclip)} class="flex-1 flex flex-col">
-          <Form.text_area form={f} field={:body} phx-debounce="300" />
-        </div>
-        <div :if={@selected_dataclip} class="flex-1 flex flex-col gap-4">
+        <div class="flex-1 flex flex-col gap-4">
           <div>
             <div class="flex flex-row">
               <div class="basis-1/2 font-semibold text-secondary-700">
                 Dataclip Type
               </div>
               <div class="basis-1/2 text-right">
-                <Common.dataclip_type_pill dataclip={@selected_dataclip} />
+                <Common.dataclip_type_pill dataclip={
+                  @selected_dataclip || %{id: "", body: [], type: :saved_input}
```

Comment:
Hmmm, I'm not sure I'm digging this interface the `dataclip_type_pill` uses...

Lets chat later today and rearrange things (including the surrounding logic), our markup a little too complicated at the moment.